_content/articles/the-sal-an-alternative-buddhist-holy-tree_ireland-John-D.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Sal — An Alternative Buddhist Holy Tree?"
+authors:
+  - "John D. Ireland"
+external_url: "https://journal.equinoxpub.com/BSR/article/view/14508/16725"
```

Comment:
This viewer is showing the entire issue, not just this article. So, I think just the DOI in `source_url` and the drive pdf link below should be enough.

Go ahead and erase this line.